Monika After Story/game/zz_backgrounds.rpy

```diff
@@ -2532,6 +2548,19 @@ init 800 python:
     #Just set us to the normal room here
     mas_setBackground(mas_background_def)
 
+init python:
+    def mas_getBackground(background_id, default=None):
+        """
+        Gets a MASFilterableBackground by id
```

Comment:
Can you add a comment stating that this needs to be at this init level so it can be used by greetings?